milli/src/search/facet/facet_distribution_iter.rs

```diff
@@ -44,7 +46,99 @@ where
     }
 }
 
-struct FacetDistribution<'t, CB>
+pub fn count_iterate_over_facet_distribution<'t, CB>(
+    rtxn: &'t heed::RoTxn<'t>,
+    db: heed::Database<FacetGroupKeyCodec<ByteSliceRefCodec>, FacetGroupValueCodec>,
+    field_id: u16,
+    candidates: &RoaringBitmap,
+    mut callback: CB,
+) -> Result<()>
+where
+    CB: FnMut(&'t [u8], u64, DocumentId) -> Result<ControlFlow<()>>,
+{
+    #[derive(Debug, PartialOrd, Ord, PartialEq, Eq)]
```

Comment:
LevelEntry's PartialOrd implementation is used in the `BinaryHeap` to compute the highest entry.

This means that the order of the fields of `LevelEntry` are significant.
Should we maybe either:

1. Document this fact?
2. Use an explicit PartialOrd implementation to break this implicit dependence to field order?